### PR TITLE
Add libgpiod

### PR DESCRIPTION
**Pull request by ggardet**
- Verification run: 
  * aarch64 tw: https://openqa.opensuse.org/t957516
  * x86_64 tw: https://openqa.opensuse.org/tests/957564
  * ppc64le tw: https://openqa.opensuse.org/tests/957567

### CODE REVIEW

File: tests/console/libgpiod.pm [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+sub run {
+    select_console 'root-console';
+
+    # Install libgpiod tools
+    zypper_call 'in libgpiod';
+
+    # Record libgpiod version
+    record_info('Version', script_output('gpiodetect --version | head -n1'));
+
+    my $gpiochipX = "gpiochip0";
```
Comment (asmorodskyi) [inline tests/console/libgpiod.pm]:
```suggestion
    my $gpiochipX = 'gpiochip' . get_var('ARCH') =~ /aarch64/ ? '1' : '0' ;
```

**Comment by asmorodskyi** [comment]
generallyy LGTM , suggestion is some minor thing which might be ignored

File: tests/console/libgpiod.pm (hunk 1 above)
Comment (ggardet) [reply tests/console/libgpiod.pm]: Updated. Thanks for the review.

**Comment by DimStar77** [comment]
Fails to pass on TW:

```
# wait_serial expected: gpiochip0 [gpio-mockup-A] (32 lines)

# Result:
gpiochip0 [gpio-mockup-A] (32 lines)
````

https://openqa.opensuse.org/tests/964423#step/libgpiod/13

**Comment by ggardet** [comment]
> Fails to pass on TW:
> 
> ```
> # wait_serial expected: gpiochip0 [gpio-mockup-A] (32 lines)
> 
> # Result:
> gpiochip0 [gpio-mockup-A] (32 lines)
> ```
> 
> https://openqa.opensuse.org/tests/964423#step/libgpiod/13

WTF? It passed on my x86_64 TW test and the string match. @okurz , any idea?

**Comment by okurz** [comment]
I just know that from my experience the regex matches can sometimes return false even though the returned output looks exactly like what is expected. I can only suggest to try to make the regex more general, e.g. just look for "gpio.*32 lines" maybe?

**Comment by ggardet** [comment]
@okurz another strange failure: https://openqa.opensuse.org/tests/965145#step/libgpiod/38 `1` does not match `1`...

**Comment by ggardet** [comment]
It should be fixed by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7768

**Pull request merged**